docs/sources/administration/plugin-management/index.md

```diff
@@ -149,6 +149,19 @@ Grafana Cloud handles the plugin installation automatically.
 
 If you're logged in to Grafana Cloud when you add a plugin, log out and then log back in again to use the new plugin.
 
+
+
+### Install plugins on a self-hosted Grafana through helm
+
+In [Helm charts](https://github.com/grafana/helm-charts/blob/79a0101296b0065b18987321b3047b8c94bc1a33/charts/grafana/values.yaml#L605), you may add the plugins you wish to install as a list
```

Comment:
I think it would be more safe to link to the configuration section of the README rather than a line in the default values file. It is slightly less convenient though.

https://github.com/grafana/helm-charts/tree/main/charts/grafana#configuration
```suggestion
With the Grafana Helm chart, add the plugins you want to install as a list using the `plugins` field in the your values file. For more information about the configuration, refer to [the Helm chart configuration reference](https://github.com/grafana/helm-charts/tree/main/charts/grafana#configuration).
```